tools/meteor-services/stats.js

```diff
@@ -37,6 +37,9 @@ var packageList = function (projectContext) {
 // - site: If it's a deploy, the name of the site ("foo.meteor.com") that we're
 //   deploying to.
 var recordPackages = function (options) {
+  if (process.env.METEOR_PACKAGE_STATS_DISABLE == "true")
```

Comment:
Other env checks here don't check for `true`, just for presence, we should probably do the same for consistency.

But also, it appears that if we include `package-stats-opt-out` package (see the code below), we will achieve the same result, so do we even need this patch? The less changes vs upstream, the better.